### ISSUE TITLE
Update inc::Devel::CheckLib version for Cygwin

### ISSUE THREAD

**Issue by tzccinct** [toddr/XML-Parser#86]
Hello.

I tried to install XML::Parser 2.46 in Cygwin perl, but it failed when executing `perl Makefile.PL`. XML::Parser 2.46 now exits in Makefile.PL when expat library and header are not found by inc::Devel::CheckLib::check_lib(). But in Cygwin, Devel::CheckLib 0.99 often fails to find expat library and header even if expat-devel package is correctly installed. I have confirmed Devel::CheckLib 1.03 or higher does not have this issue. Would you update inc::Devel::CheckLib version from 0.99 to 1.03 or higher?

For information, no problem on Devel::CheckLib 0.99 if you install libnsl-devel package on Cygwin, but I suppose libnsl-devel package is not in Cygwin base packages so not so common. Devel::CheckLib 1.03 or higher ignores -lnsl ldflags when checking expat library/header and no problem in any case.

Thanks.

**Comment by tzccinct** [comment]
```
$ cygcheck -c libexpat-devel
Cygwin Package Information
Package              Version        Status
libexpat-devel       2.2.6-1        OK

$ cygcheck -l libexpat-devel
/usr/include/expat.h
/usr/include/expat_config.h
/usr/include/expat_external.h
/usr/lib/libexpat.a
/usr/lib/libexpat.dll.a
/usr/lib/pkgconfig/expat.pc

$ LANG=C cpanm -v XML::Parser
cpanm (App::cpanminus) 1.7044 on perl 5.032001 built for x86_64-cygwin-threads-multi
Work directory is /home/tzccinct/.cpanm/work/1614478084.6378
You have make /usr/bin/make
You have LWP 6.52
You have /usr/bin/tar: tar (GNU tar) 1.34
Packaged by Cygwin (1.34-1)
Copyright (C) 2021 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by John Gilmore and Jay Fenlason.
Searching XML::Parser () on cpanmetadb ...
--> Working on XML::Parser
Fetching http://www.cpan.org/authors/id/T/TO/TODDR/XML-Parser-2.46.tar.gz ... OK
Unpacking XML-Parser-2.46.tar.gz
XML-Parser-2.46/
XML-Parser-2.46/Parser/
XML-Parser-2.46/Parser/Encodings/
XML-Parser-2.46/Parser/Encodings/iso-8859-2.enc
XML-Parser-2.46/Parser/Encodings/iso-8859-7.enc
XML-Parser-2.46/Parser/Encodings/iso-8859-5.enc
XML-Parser-2.46/Parser/Encodings/iso-8859-15.enc
XML-Parser-2.46/Parser/Encodings/x-sjis-unicode.enc
XML-Parser-2.46/Parser/Encodings/windows-1250.enc
XML-Parser-2.46/Parser/Encodings/x-euc-jp-unicode.enc
XML-Parser-2.46/Parser/Encodings/iso-8859-9.enc
XML-Parser-2.46/Parser/Encodings/windows-1251.enc
XML-Parser-2.46/Parser/Encodings/iso-8859-8.enc
XML-Parser-2.46/Parser/Encodings/windows-1255.enc
XML-Parser-2.46/Parser/Encodings/iso-8859-4.enc
XML-Parser-2.46/Parser/Encodings/x-euc-jp-jisx0221.enc
XML-Parser-2.46/Parser/Encodings/koi8-r.enc
XML-Parser-2.46/Parser/Encodings/iso-8859-3.enc
XML-Parser-2.46/Parser/Encodings/Japanese_Encodings.msg
XML-Parser-2.46/Parser/Encodings/ibm866.enc
XML-Parser-2.46/Parser/Encodings/windows-1252.enc
XML-Parser-2.46/Parser/Encodings/euc-kr.enc
XML-Parser-2.46/Parser/Encodings/README
XML-Parser-2.46/Parser/Encodings/big5.enc
XML-Parser-2.46/Parser/Encodings/x-sjis-jisx0221.enc
XML-Parser-2.46/Parser/Encodings/x-sjis-jdk117.enc
XML-Parser-2.46/Parser/Encodings/x-sjis-cp932.enc
XML-Parser-2.46/Parser/Style/
XML-Parser-2.46/Parser/Style/Objects.pm
XML-Parser-2.46/Parser/Style/Stream.pm
XML-Parser-2.46/Parser/Style/Debug.pm
XML-Parser-2.46/Parser/Style/Subs.pm
XML-Parser-2.46/Parser/Style/Tree.pm
XML-Parser-2.46/Parser/LWPExternEnt.pl
XML-Parser-2.46/Expat/
XML-Parser-2.46/Expat/typemap
XML-Parser-2.46/Expat/encoding.h
XML-Parser-2.46/Expat/Expat.pm
XML-Parser-2.46/Expat/Expat.xs
XML-Parser-2.46/Expat/Makefile.PL
XML-Parser-2.46/t/
XML-Parser-2.46/t/skip.t
XML-Parser-2.46/t/namespaces.t
XML-Parser-2.46/t/stream.t
XML-Parser-2.46/t/partial.t
XML-Parser-2.46/t/file_open_scalar.t
XML-Parser-2.46/t/decl.t
XML-Parser-2.46/t/ext.ent
XML-Parser-2.46/t/foo.dtd
XML-Parser-2.46/t/finish.t
XML-Parser-2.46/t/cdata.t
XML-Parser-2.46/t/styles.t
XML-Parser-2.46/t/external_ent.t
XML-Parser-2.46/t/parament.t
XML-Parser-2.46/t/ext2.ent
XML-Parser-2.46/t/file.t
XML-Parser-2.46/t/astress.t
XML-Parser-2.46/t/encoding.t
XML-Parser-2.46/t/defaulted.t
XML-Parser-2.46/Makefile.PL
XML-Parser-2.46/samples/
XML-Parser-2.46/samples/REC-xml-19980210.xml
XML-Parser-2.46/samples/xmlfilter
XML-Parser-2.46/samples/canontst.xml
XML-Parser-2.46/samples/xmlcomments
XML-Parser-2.46/samples/ctest.dtd
XML-Parser-2.46/samples/canonical
XML-Parser-2.46/samples/xmlstats
XML-Parser-2.46/MANIFEST
XML-Parser-2.46/inc/
XML-Parser-2.46/inc/Devel/
XML-Parser-2.46/inc/Devel/CheckLib.pm
XML-Parser-2.46/Changes
XML-Parser-2.46/README
XML-Parser-2.46/Parser.pm
XML-Parser-2.46/META.yml
XML-Parser-2.46/META.json
Entering XML-Parser-2.46
Checking configure dependencies from META.json
Checking if you have ExtUtils::MakeMaker 6.58 ... Yes (7.44)
Running Makefile.PL
Configuring XML-Parser-2.46 ...
Expat must be installed prior to building XML::Parser and I can't find
it in the standard library directories. Install 'expat-devel' (or
'libexpat1-dev') package with your OS package manager. See 'README'.

Or you can download expat from:

http://sourceforge.net/projects/expat/

If expat is installed, but in a non-standard directory, then use the
following options to Makefile.PL:

    EXPATLIBPATH=...  To set the directory in which to find libexpat

    EXPATINCPATH=...  To set the directory in which to find expat.h

For example:

    perl Makefile.PL EXPATLIBPATH=/home/me/lib EXPATINCPATH=/home/me/include

Note that if you build against a shareable library in a non-standard location
you may (on some platforms) also have to set your LD_LIBRARY_PATH environment
variable at run time for perl to find the library.

N/A
! Configure failed for XML-Parser-2.46. See /home/tzccinct/.cpanm/work/1614478084.6378/build.log for details.
```

**Comment by toddr** [comment]
This was fixed in #89
